src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiParser.cs

```diff
@@ -128,6 +125,8 @@ public void Parse(string message)
             }
         }
 
+        private static bool IsDigit(char c) => (uint)((int)c - (int)'0') <= (uint)('9' - '0');
```

Comment:
please mark this with [MethodImpl(MethodImplOptions.AggressiveInlining)]

Reply:
what does this do?

Reply:
I am expecting the compiler to inline this method (as it is small one) to avoid the cost of calling a method. Adding the attribute may help forcing that if somehow the compiler didn't inline it by default.

Reply:
This method is tiny and should be inlined by the JIT.

The JIT uses heuristics to determine whether a method should be inlined or not. With the suggested attribute we give the JIT a hint to bypass some heuristics and "force" to inline the method.

So the benefit is
* less work for the JIT
* to be on the safer side, that it is actually inlined